### ISSUE TITLE
Subscriptions: Forms should support csrf.mode: ajax

### ISSUE THREAD

**Issue by michnhokn** [mollie/Shopware6#419]
Currently using the mollie plugin with [`csrf.mode: ajax`](https://developer.shopware.com/docs/guides/plugins/plugins/storefront/use-csrf-protection#use-csrf-protection-for-form) breaks the async actions from the account.

An unsubscribe or change of the sipping address is then no longer possible.

The fix for this problem is the mode `ajax` for CSRF tokens. In this mode Shopware is fetching a token using a XML HTTP request before sending a POST request. Enabling this behavior in our plugins is quite easy. If you're dealing with a `form` element, simply add the JS plugin `data-form-csrf-handler="true"` to it and you're good to go.

If you're fetching data using a POST request in your JavaScript, please use the method `fetchCsrfToken` from the HTTP client to fetch a token before sending the actual request:
``` javascript
this.client.fetchCsrfToken((token) => {
   // console.log(token);
})
```

**Comment by boxblinkracer** [comment]
Hi
sry for the long delay

that totally makes sense
ill make sure its developed

thanks for this

**Comment by boxblinkracer** [comment]
released in v3.2.0
